config/templates/etc/filebeat/filebeat.yml.j2

```diff
@@ -258,3 +258,9 @@ logging.json: true
 
 # This allows to enable 6.7 migration aliases
 #migration.6_to_7.enabled: true
+
+#=========================== External Inputs ===================================
+filebeat.config.inputs:
+  enabled: true
```

Comment:
Just a small details, could this be disabled by default and be enabled by choice from users only please? Thanks!

Reply:
Disabled by default (var beats_config.filebeat.enable_external_inputs)